Pod/Native/NativeRunner.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  NativeRunner.swift
+//  Pods
+//
+//  Created by jacdevos on 2016/11/15.
+//
+//
+
+import Foundation
+import XCTest
+
+//Gives us the ability to run features or scenarios directly by specifying file and name
+open class NativeRunner {
+    
+    public class func runScenario(featureFile: String, scenario: String?, testCase: XCTestCase) {
+        testCase.state.loadAllStepsIfNeeded()
+
+        guard let path = (Bundle(for: type(of: testCase)).resourceURL?.appendingPathComponent(featureFile)) else {
+            XCTFail("Path could not be built for feature file: \(featureFile)")
+            return
+        }
+        
+        let features = loadFeatures(path: path)
+        
+        for feature in features {
+            let scenarios = feature.scenarios.filter({
+                scenario == nil || $0.scenarioDescription.hasPrefix(scenario!)
+            })
+            
+            if scenarios.count < 1 {
+                XCTFail("No scenario found with name: \(scenario)")
+            }
+            
+            for scenario in scenarios {
```

Comment:
Good suggestion @smaljaar - maybe create another PR for this, I see change has been merged.